examples/policies/l3/host/host-policy.json

```diff
@@ -0,0 +1,41 @@
+[
+  {
+    "nodeSelector": {"matchLabels": {"doh":"no"}},
```

Comment:
Do we have a way not to put the `matchLabels` in the file, or are they mandatory?

Reply:
Not sure if that's what you're asking, but if you want to match all nodes regardless of labels, you can do:
```json
    "nodeSelector": {}
```

Reply:
The idea was that if we can remove the `"doh":"no"` from that example, maybe we should. Not a big concern, but this is the kind of example file that users will tend to joyfully copy-paste from, and we'll end up with plenty of `"doh":"no"` in the wild :)

Reply:
In case that wasn't clear, "doh" here is not Homer Simpson's expression but short for DNS-over-HTTPS. This policy whitelists a couple needed ports for Cilium, then whitelists DNS, but not DOH. That was the point of my short host firewall demo a while back.
I will likely update this example and incorporate it in a getting started guide once I write the documentation followup.